apps/squeaknode/docker-compose.yml

```
@@ -0,0 +1,42 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: yzernik/squeaknode:v0.1.21@sha256:8555988fd28b4d9e6533827510f8f618182d10de3f6e06b988d89048617615ba
```

The container isn't stopping when you run `scripts/app stop squeaknode`. It just runs for the 1m for the `stop_grace_period` and then gets killed. Probably because signals aren't being passed through, wrapping in a mini init system should resolve it.

```suggestion
    image: yzernik/squeaknode:v0.1.21@sha256:8555988fd28b4d9e6533827510f8f618182d10de3f6e06b988d89048617615ba
    init: true
```

---

I did some debugging and found the orphan thread that was stopping the clean shutdown. I think I can skip the init option.